Conform to ASWebAuthenticationPresentationContextProviding protocol

jwelker110 opened the pull request:
Fixes #1

jwelker110 committed:
Conform to ASWebAuthenticationPresentationContextProviding protocol

rak13 commented:
Thanks for the update.
I guess I should not merge this PR as it creates a new Package for publishing

jwelker110 committed:
Create new separate package for publishing

jwelker110 commented:
Thanks for the heads up. I moved those changes to a separate branch so they won't affect this PR.

jwelker110 committed:
Wait to call start until Application becomes active

jwelker110 committed:
Merge branch 'master' into feature/new-package-fixed

jwelker110 committed:
Bump version

jwelker110 committed:
Update changelog and versions

pizzapanther committed:
Update ASWebAuthSession.js

jwelker110 committed:
Merge branch 'master' into master

jwelker110 committed:
Merge pull request #1 from pizzapanther/master

Namespace exec call

jwelker110 committed:
Version bump. Changelog update

Commit:
fix(varname): rename session variable

Will avoid duplicate symbols error
when used with cordova-plugin-sfauthenticationsession

jwelker110 committed:
Merge branch 'SDA-SE-master'

jwelker110 committed:
Update to 1.1.4

jwelker110 committed:
Update README.md

No longer maintaining
